PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -325,12 +322,9 @@ using namespace pat;
 using namespace std;
 
 PATMuonHeavyObjectCache::PATMuonHeavyObjectCache(const edm::ParameterSet& iConfig) {
-  if (iConfig.getParameter<bool>("computeMuonMVA")) {
-    edm::FileInPath mvaTrainingFile = iConfig.getParameter<edm::FileInPath>("mvaTrainingFile");
-    edm::FileInPath mvaLowPtTrainingFile = iConfig.getParameter<edm::FileInPath>("lowPtmvaTrainingFile");
+  if (iConfig.getParameter<bool>("computeMiniIso")) {
     float mvaDrMax = iConfig.getParameter<double>("mvaDrMax");
-    muonMvaEstimator_ = std::make_unique<MuonMvaEstimator>(mvaTrainingFile, mvaDrMax);
-    muonLowPtMvaEstimator_ = std::make_unique<MuonMvaEstimator>(mvaLowPtTrainingFile, mvaDrMax);
+    calculatePtRatioRel_ = std::make_unique<CalculatePtRatioRel>(mvaDrMax);
```

Comment:
Ok, I will update it. I had just put `dr` because that is how it was in the code was being used (`MuonMvaEstimator.cc`). I have just changed the name of the file and removed the code related with the MVAs (the rest I left as it was).